Stripe/PublicHeaders/STPRedirectContext.h

```diff
@@ -60,8 +60,11 @@ typedef STPRedirectContextSourceCompletionBlock STPRedirectContextCompletionBloc
 
 /**
  A callback that is executed when the context believes the redirect action has been completed.
+ 
+ @note The STPPaymentIntent originally provided to this class may be out of date,
+ so you should re-fetch it using the clientSecret.
```

Comment:
👍  Love it. I'm pretty sure this is *why* the client secret is being provided, but I don't think I made that explicit.